framework/scripts/get_repo_revision.py

```diff
@@ -204,7 +204,8 @@ def writeRevision(self, repo_location, app_name, revision_header):
             timestamps_to_modify.append(os.path.join(cdup, '.git', 'index'))
 
             for touch_file in timestamps_to_modify:
-                shellCommand("touch -r " + revision_header + " " + touch_file, repo_location)
+                if os.path.isfile(touch_file):
```

Comment:
Is this going to be the correct path? This will be relative to current working directory and not `repo_location`

Reply:
Nope - it won't be the right path...